closed/OpenJ9.gmk

```diff
@@ -298,7 +306,7 @@ $(OUTPUT_ROOT)/vm/cmake.stamp :
 build-j9vm : $(OUTPUT_ROOT)/vm/cmake.stamp generate-j9-version-headers generate-j9jcl-sources
 	@$(ECHO) "Compiling OpenJ9 in $(OUTPUT_ROOT)/vm"
 	$(call ShowVersions)
-	+export OPENJ9_BUILD=true $(CUSTOM_COMPILER_ENV_VARS) \
+	+export $(CUSTOM_COMPILER_ENV_VARS) \
 		&& $(MAKE_VM) $(MAKE_ARGS) -C $(OUTPUT_ROOT)/vm JAVA_VERSION=80 VERSION_MAJOR=8 all
```

Comment:
`CUSTOM_COMPILER_ENV_VARS` may be empty in which case this becomes `export && ...` which prints all environment variables (which we don't want). We can remove `export` and `&&` like other versions:
```
	+$(CUSTOM_COMPILER_ENV_VARS) \
		$(MAKE_VM) $(MAKE_ARGS) -C $(OUTPUT_ROOT)/vm JAVA_VERSION=80 VERSION_MAJOR=8 all
```

Reply:
Updated.